### PR TITLE
docs(README): add 'yarn build dev' to dev steps

### PR DESCRIPTION
**Pull request by rubinovitz**
## PR Description
Dev instructions in README did not include building project.

### Testing
- [ ] added relevant test coverage
- [x] no tests needed

### Docs
- [ ] added relevant docs
- [x] updated relevant sections in the README.md
- [ ] updated relevant docstrings in index.d.ts
- [ ] no docs needed

### Typescript
- [ ] added or edited relevant Typescript type declarations
- [x] no type declaration updates needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -210,6 +210,7 @@ https://www.notion.so/Homebase-Alpha-Docs-0f0e22f3adcd4e9d87a13440ab0c7a0b
 
 ```bash
 yarn install
+yarn build dev
```
Comment (smothers) [inline README.md:213]: why is this needed?

also, did you mean `yarn build:dev` or `yarn build`?

Comment (rubinovitz) [reply README.md:213]: I've been using `yarn build dev`, but `yarn build` might be sufficient.

I need this because otherwise dist doesn't get built and I get a bunch of errors when I open localhost:3000

Comment (rubinovitz) [reply README.md:213]: I'll dig into it more, this might not have been what fixed the issue.

Comment (smothers) [reply README.md:213]: You shouldn't need dist to run the dev server... And yarn test already runs yarn build

Comment (rubinovitz) [reply README.md:213]: gotcha, I'll close this then and next time this happens I'll try to figure out what is breaking the build. Might be a caching thing. Thanks!
